ref/general/what-is-open-liberty.adoc

```diff
@@ -0,0 +1,122 @@
+Open Liberty monitors the server XML config for updates and dynamically reloads whenever changes are detected.
+For more information, see link:/docs/ref/config/#serverConfiguration.html[Server Configuration].
+
+== Ready for production
+
+More than just an intuitive development environment, Open Liberty is battle-hardened and ready for whatever production scenario your applications require.
+Zero-migration architecture, automated performance tuning, and easy integration with other open-source software help you deploy and maintain secure, highly available applications.
+
+=== Zero-migration architecture
+
+With the Open Liberty zero-migration architecture, you focus on writing your applications, not the APIs changing under you.
+Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty, without unexpected changes in behavior.
+Open Liberty offers pluggable features and full compatibility between product versions so that your existing APIs and behaviors are supported in each new release.
+
+Open Liberty does not modify user configuration files, which are fully compatible between versions.
+You can use a single version of your configuration files across multiple versions.
+Files that you created for a previous version of Open Liberty can be used with a later version.
+Files that you create for later versions can be used with previous versions.
+As a result, if all configured features are installed, you can use a single set of configuration files across multiple versions without modifications.
+Any configuration settings that do not apply to a particular version of Open Liberty are ignored.
+
+=== Dependable security from development to Day 2, and beyond
+
+Open Liberty supports application security with a robust set of security features.
+During development and testing, Open Liberty's built-in link:/docs/ref/general/#basic-registry.html[basic user registry and the QuickStart security element] offer easily configurable security frameworks to test access to your application in a lightweight, self-contained manner.
+The `securityUtility` supports plain text encryption, so you can easily secure user passwords with hash encoding.
+
+After deployment, the link:/docs/ref/feature/#appSecurity.html[Application Security] feature provides the core capability for the Jakarta EE Security API, while the link:/docs/ref/feature/#transportSecurity.html[Transport Security] feature enables secure TLS connections by using HTTPS.
+The link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] feature supports authentication and authorization from external LDAP registries.
+Open Liberty also provides features to support a range of single sign-on options, including link:/docs/ref/feature/#jwtSso.html[JSON Web Tokens], link:/docs/ref/feature/#jwtSso.html[OpenID Connect], link:/docs/ref/feature/#samlWeb.html[SAML web], and link:/docs/ref/feature/#spnego.html[SPNEGO].
```

Comment:
I think "web" should be capitalized in "SAML web". There's an issue open in the OL repo to update the feature name in the autogen.